README.md

```
@@ -97,6 +100,22 @@ YOUTUBE_COOKIES_BASE64="base64-encoded-cookies-content"
 # YOUTUBE_COOKIES="/path/to/cookies.txt"
 ```
 
+`ELEVENLABS_WEBHOOK_ID` と `ELEVENLABS_WEBHOOK_SECRET` が両方設定されている
+場合、Bot からの文字起こしは非同期 Webhook に切り替わります。未設定の場合と
+`scribe` CLI は従来どおり同期処理を使います。
+
+ElevenLabs の Speech-to-Text Webhook URL には次を設定します：
+
+```
```

markdownlint-cli2 (0.23.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)